pkg/controller.v1/common/job_controller.go

```diff
@@ -69,9 +69,11 @@ var (
 type GangScheduler string
 
 const (
-	GangSchedulerNone             GangScheduler = "None"
+	GangSchedulerNone             GangScheduler = ""
```

Comment:
What intent of using an empty string instead of `None`?

Reply:
if GangSchedulerNone is None, It may be considered as enabling the gang scheduling, but I think we can remove the GangSchedulerNone const.
```
var GenNonGangSchedulerSetupFunc = func() GangSchedulingSetupFunc {
	return func(jc *JobController) {
		jc.Config.GangScheduling = GangSchedulerNone
		jc.PodGroupControl = nil
	}
}

func (c *JobControllerConfiguration) EnableGangScheduling() bool {
	return c.GangScheduling != ""
}
```

Reply:
Thanks for clarifying.

>  but I think we can remove the GangSchedulerNone const.

Sounds good. However, it might be better to do that after we merge kubeflow/common into kubeflow/training-operator since GangSchedulerNone might be worth it as documentation.

Reply:
> since GangSchedulerNone might be worth it as documentation.

Make sense, let's keep it.

Reply:
Yes, the "None" was already released on v0.4.6.

Reply:
Yes, we should keep it for backwards compatibility.